fv_be/app/models/part_of_speech.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+# FirstVoices
+from .base import BaseModel
+
+
+class PartOfSpeech(BaseModel):
+    """Model for Categories."""
+
+    # Fields
+    # todo: name for the following field, can we put title ? or will it be very similar to label ?
+    # since other models have a title field
+    name = models.CharField(
+        max_length=200
+    )  # from wiki: name of the part of speech in snake case e.g.intransitive_verb
+    label = models.CharField(max_length=200)
```

Comment:
Do they need to have both a name and a label? Seems like we could use an ID from the base model as the primary key, ditch the snake_case label from nuxeo, and just keep the formatted one as a title.

Reply:
I would agree with you @caforbes. And maybe when we only have one field, we can name it 'title' to make it consistent with the rest of the models ?
Tagging @gmcauliffe for his view about doing away with nuxeo snake_case lables.

Reply:
Just remember that it will go through i18n before being displayed, so the db value doesn't need to be especially well formatted